packages/client/hooks/useTrebuchetEvents.ts

```diff
@@ -63,6 +63,7 @@ const useTrebuchetEvents = () => {
       // hacky but that way we don't have to double parse huge json payloads. SSE graphql payloads are pre-parsed
       if (typeof payload !== 'string' || !payload.startsWith('{"version":')) return
       const obj = JSON.parse(payload)
+      if (obj.version) atmosphere.version = obj.version
```

Comment:
nit: i'd name this `serverVersion`
then again, if we fix the service worker, with the above code change, the code someone runs should never be stale which means we don't have to do any of that error handling!

can you verify?

here's how i'd test it (after making the above fix):
1. remove any existing serviceWorkers from the browser under localhost
2. in the server mutation "createTask" write `console.log(newTask.threadSortOrder)` 
3. run `yarn build` and then `yarn start`
4.  create a new task & verify that the server is logging the thread sort order
5. change the CreateTaskMutation client mutation so that `variables.newTask.threadSortOrder = 999`
6. change the version in `handleOpen` to something else
7. run `yarn build` and then `yarn start`. without refreshing the page, the client should reconnect & pop the snackbar.
8. create a task & verify that the server receives 999 as the threadSortOrder

if all of those work, then no error case is needed!

Reply:
That worked! I've pushed the update and removed the error handling from the `ErrorComponent`